ansible-playbook-base/roles/prepare-gluster-server/tasks/centos.yml

```diff
@@ -18,6 +18,19 @@
   tags:
     - packagesall
     - packagesserver
+  with_items:
```

Comment:
Item list is not used in task, so how is this working? Will be testing it and providing feedback on the same.

Reply:
Tested, the list is not used, as a result it tries and installs gluster-server 'n' times, 'n' being the list lenght.

Further, to make this work we need a specific loop variable (currently it defaults to item, and that is incorrect, as main.yml already is using the same). If we need to install individual packages, we would need to add loop_control and var as follows here,
```
loop_control:
    loop_var: plist_item
```